emscripten.py

```diff
@@ -840,7 +840,7 @@ def create_module(sending, receiving, invoke_funcs, metadata):
   module = []
 
   module.append('var asmLibraryArg = %s;\n' % sending)
-  if settings.ASYNCIFY and settings.ASSERTIONS:
+  if settings.ASYNCIFY:
```

Comment:
I guess this was just for debugging?

Reply:
Ah, this is indeed wrong, but it is for more than debugging. I'll fix + add a comment. We now need to instrument imports for either stack switching (to use the Promise API) or for assertions (to add asserts on proper import handling).